### DISCUSSION TITLE
Copilot interferes with IntelliJ's auto completion

### DISCUSSION THREAD

**Discussion by TheDome0** [community/community#14286] [Copilot Conversations] [answered]
Copilot seems to suppress the code completion IntelliJ offers.

Without Copilot
![image_2022-04-04_211833](https://user-images.githubusercontent.com/85122272/161616174-42a203af-f689-4e17-b002-c378569fb5ab.png)

With Copilot enabled
![image](https://user-images.githubusercontent.com/85122272/161616196-5ccc21f9-7ddc-471a-b4ba-fb3670693e78.png)

in this example it ended up suggesting exactly what I needed but that's because I've already written similar tests. In the first ones I had to laboriously write the name of my service class by hand since copilot prevented me from using IntelliJ's code completion.

**Comment by jansorg** [comment, marked as answer]
There's a setting to show both side-by-side at `Preferences > Languages & Frameworks > GitHub Copilot`.

**Reply by HreshchyshynT** [reply]
That should be enabled by default. I disabled copilot because I didn't know about this feature and sometimes I needed just Intelij to autocomplete function/class name. Thanks a lot, copilot got a second breath now 🔥

**Reply by brokenthorn** [reply]
My God! This had me going mad thinking WebStorm was bugged or the LSP or... 💢

Thank you!
